android/app/src/main/res/values/strings.xml

```diff
@@ -190,4 +191,25 @@ liability.</p> ]]></string>
     <string name="privacy_policy_text"><![CDATA[To proceed please accept the <a href="https://www.jetbrains.com/company/privacy.html">Privacy Policy</a>]]></string>
     <string name="privacy_policy_accept">I accept the Privacy Policy</string>
     <string name="submit_button">Submit</string>
+
+    <!--Basic user location permission toasts-->
+    <string name="user_location_permission_explanation">Please enable location permissions to view your device\'s location.</string>
+    <string name="user_location_permission_not_granted">You didn\'t grant location permissions.</string>
+
+    <!-- Mapbox map access token
```

Comment:
Do we need it for testing? It makes it harder to contribute. 
Also adding own string resource is not really intuitive. What about adding resource in build.gradle just like we do with API_URL:
buildConfigField("String", "API_URL", "\"https://api.kotlinconf.com\"")
Only instead of actual value reference local property resource or fake token if property not present.

Reply:
Ok, added to the gradle file:

```groovy
local {
            buildConfigField("String", "API_URL", "\"https://api.kotlinconf.com\"")
            buildConfigField("String", "MAPBOX_ACCESS_TOKEN", MAPBOX_ACCESS_TOKEN)
            dimension "server"
        }
prod {
            buildConfigField("String", "API_URL", "\"https://api.kotlinconf.com\"")
            buildConfigField("String", "MAPBOX_ACCESS_TOKEN", MAPBOX_ACCESS_TOKEN)
            dimension "server"
}
```

```
Mapbox.getInstance(this, BuildConfig.MAPBOX_ACCESS_TOKEN)
```

Can confirm that map loading works with this config. I put `MAPBOX_ACCESS_TOKEN` in the `gradle.properties` file which lives in my computer user profile level `.gradle` folder.